cmake/install_codesign_script.cmake

```diff
@@ -113,7 +113,7 @@ foreach(path ${_all_root_dylibs})
   endif()
 endforeach()
 
-file(GLOB PYTHON_SOS "${CMAKE_INSTALL_PREFIX}/python_lib/lib-dynload/*.so")
+file(GLOB PYTHON_SOS "${CMAKE_INSTALL_PREFIX}/python_lib/lib-dynload/*.so" "${CMAKE_INSTALL_PREFIX}/python_lib/lib-dynload/*.dylib")
```

Comment:
And when we glob the python .sos, we also glob the tcl and tk dylibs so they get signed as well.

```
/Users/julien/Software/Others/EnergyPlus-build-release/_CPack_Packages/Darwin/TGZ/EnergyPlus-24.2.0-cdf84ccb68-Darwin-macOS12.1-arm64/energyplus auxiliary eplaunch
```

![image](https://github.com/user-attachments/assets/3803c7eb-fb9e-4a3f-a494-f67c5235e721)


the end.